src/coreclr/vm/jithelpers.cpp

```diff
@@ -1484,6 +1485,47 @@ HCIMPL0(void, IL_Rethrow)
 }
 HCIMPLEND
 
+HCIMPL1(void, IL_ThrowExact, Object* obj)
+{
+    FCALL_CONTRACT;
+
+    /* Make no assumptions about the current machine state */
+    ResetCurrentContext();
+
+    FC_GC_POLL_NOT_NEEDED();    // throws always open up for GC
+
+    OBJECTREF oref = ObjectToOBJECTREF(obj);
+    GetThread()->GetExceptionState()->SetRaisingForeignException();
+
+#ifdef FEATURE_EH_FUNCLETS
+    Thread *pThread = GetThread();
+    
+    SoftwareExceptionFrame exceptionFrame;
+#ifdef TARGET_X86
+    _ASSERTE(!"Introduce `IL_ThrowExact_x86` following the pattern of other Throw helpers.");
+    // exceptionFrame.UpdateContextFromTransitionBlock(transitionBlock);
```

Comment:
Merging main and cherry-picking https://github.com/dotnet/runtimelab/commit/ad964673f457fde7bed7634fad7a7f9644a2edf8 would resolve it.

Reply:
This has been resoved now.